recipe/meta.yaml

```diff
@@ -16,9 +16,10 @@ source:
     - patches/0001-do-not-link-msvc-runtime-statically.patch
     - patches/0002-fix-paths-for-include-lib-directories.patch
     - patches/0003-adapt-to-name-of-protobuf-lib-on-windows.patch
+    - patches/0004-fix-build-with-python-311.patch
```

Comment:
Patching with https://github.com/protocolbuffers/protobuf/pull/10403 until there's a release with that fix.